### DISCUSSION TITLE
AdonisJS v5

### DISCUSSION THREAD

**Discussion by gabbmariz** [adonisjs/core#1976] [General]
Is there already a possible release date, or is it still far from being released?

**Comment by iagobruno** [comment]
According to the official Twitter profile, they intend to take Adonis out of the preview in December.

[![20201128_173716](https://user-images.githubusercontent.com/3616259/100525541-7dcc4700-31a0-11eb-8035-5af3245a87c4.png)](https://twitter.com/adonisframework/status/1320622832175337475)

But don't wait, [Adonis v5 is stable enough to be used in production](https://github.com/adonisjs/core/discussions/1257)!

**Reply by gabbmariz** [reply]
ooooh nice, just in time, I have a project to develop, and adonisjs is excellent for that, but I don't want to use version 4.1
